src/App.js

```diff
@@ -307,94 +308,14 @@ class App extends React.Component {
-                  <div className="event-card-wrapper__timing text-base text-c200 flex items-center leading-relaxed">
-                    <span className="tracking-wide">admin</span>
-                    &nbsp; &nbsp;
-                    <span>12:00 am</span>
-                  </div>
-                  <div className="event-card-wrapper__topic">
-                    <p className="text-c100 font-bold text-lg leading-relaxed">
-                      Pure water and fresh food
-                    </p>
-                  </div>
-                  <div className="event-card-wrapper__location">
-                    <p className="text-c600 text-base flex items-center leading-relaxed">
-                      Broklyn Street 40, New York
-                    </p>
-                  </div>
-                  <div className="event-card-wrapper__date">
-                    <div className="event_date">
-                      <span> 07</span>
-                      <span> Feb</span>
-                      <span>2017</span>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div className="event-card-wrapper flex w-4/5 mb-4">
-                <div className="event-card-wrapper_image">
-                  <img src="./assets/img/events/footer-news.jpg" alt="" />
-                </div>
-                <div className="event-card-wrapper__details relative self-center pl-16">
-                  <div className="event-card-wrapper__timing text-base text-c200 flex items-center leading-relaxed">
-                    <span className="tracking-wide">admin</span>
-                    &nbsp; &nbsp;
-                    <span>12:00 am</span>
-                  </div>
-                  <div className="event-card-wrapper__topic">
-                    <p className="text-c100 font-bold text-lg leading-relaxed">
-                      Pure water and fresh food
-                    </p>
-                  </div>
-                  <div className="event-card-wrapper__location">
-                    <p className="text-c600 text-base flex items-center leading-relaxed">
-                      Broklyn Street 40, New York
-                    </p>
-                  </div>
-                  <div className="event-card-wrapper__date">
-                    <div className="event_date">
-                      <span> 07</span>
-                      <span> Feb</span>
-                      <span>2017</span>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
+            <UpcomingEvents />
 
             <div className="w-3/5 relative z-20">
               <div className="absolute z-30">
-                <img
-                  src="./assets/img/events/bg_2.png"
-                  alt="upcoming events right-bg"
-                />
+                <img src="PNG Background" alt="upcoming events right-bg" />
```

Comment:
what is this?

Reply:
This is the image in the red card .. the App.js doesn't import the src .. so i put any thing in the src